index.js

```diff
@@ -43,6 +43,13 @@ load.path = function (dir) {
   var nearby = resolve(path.dirname(process.execPath))
   if (nearby) return nearby
 
+  // append platform, and prefix with scoped name matching package name if unscoped
+  var platformPackage = (packageName[0] === '@' ? '' : '@' + packageName + '/') + packageName + '-' + platform + '-' + arch
```

Comment:
One last thing: there's a chance that `packageName` is undefined (because of the try/catch above).